<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OrderWidget</class>
 <widget class="QWidget" name="OrderWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>706</width>
    <height>358</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page_2">
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="3" column="4">
        <widget class="QToolButton" name="toolButton">
         <property name="text">
          <string>会员卡支付</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1" colspan="5">
        <widget class="QTableWidget" name="tableWidget_2">
         <property name="minimumSize">
          <size>
           <width>300</width>
           <height>0</height>
          </size>
         </property>
         <column>
          <property name="text">
           <string>菜名</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>数量</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>单价</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>id</string>
          </property>
         </column>
        </widget>
       </item>
       <item row="2" column="1" colspan="2">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>总计：</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QToolButton" name="toolButton_2">
         <property name="text">
          <string>删除</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0" rowspan="4">
        <widget class="QStackedWidget" name="stackedWidget_2">
         <property name="currentIndex">
          <number>1</number>
         </property>
         <widget class="QWidget" name="page_3">
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="margin">
            <number>0</number>
           </property>
           <item>
            <widget class="QListView" name="disheTypeList">
             <property name="viewMode">
              <enum>QListView::IconMode</enum>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="page_4">
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QToolButton" name="toolButton_5">
             <property name="text">
              <string>返回</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QListView" name="dishesList">
             <property name="viewMode">
              <enum>QListView::IconMode</enum>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item row="3" column="2" colspan="2">
        <widget class="QToolButton" name="toolButton_3">
         <property name="text">
          <string>现金收讫</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
